### PR TITLE
Canadian Championship Bidding Process initial draft.

### PR DESCRIPTION
**Pull request by tdmittens**
This will act as our policy for the Canadian Championship bidding process moving forward. Any feedback/modifications are welcome.

### CODE REVIEW

File: public/src/canadian-championship-bidding-policy.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+Canadian Championships _should_ should be conducted every 2 years, unless otherwise noted by Speedcubing Canada's Board of Directors.
+
+## Canadian Championship Selection Committee Creation
+
+- For every Canadian Championship, one member of Speedcubing Canada's Board of Directors _should_ be selected to lead a separate Canadian Championship Selection Committee.
+- The Canadian Championship Selection Committee **must** include all current members of Speedcubing Canada's Board of Directors.
+- The selected leader _may_ add additional members as a part of the Canadian Championship Selection Committee.
+  - Selected members **must** be Speedcubing Canada members at the time of selection.
+  - Selected members _should_ have previous experience in organizing large competitions.
```
Comment (sarahstrong314) [inline public/src/canadian-championship-bidding-policy.md:22]: Maybe strictly define what "large" means?

File: public/src/canadian-championship-bidding-policy.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+- Submission dates for applications must be posted by the Canadian Championship Selection Committee at least 21 days before submissions are due.
+  - The application submission date _may_ be moved with written notice from Speedcubing Canada’s Board of Directors.
+- The Canadian Championship Selection period _should_ be a minimum of 14 days.
+- The selection of a host for the Canadian Championships _should_ be concluded at least 9 months before the start of the event.
+
+## Canadian Championship Application Submission Process
+
+- A Canadian Championship Application will consist of a PDF document that contains information to support the Canadian Championship Selection Committee with the Canadian Championship Selection Process.
+- Every application **must** have a point person if an application is being submitted on behalf of a larger organization team.
```
Comment (sarahstrong314) [inline public/src/canadian-championship-bidding-policy.md:34]:
```suggestion 
Applications that are being submitted on behalf of a larger organization team **must** have a point person.
```

File: public/src/canadian-championship-bidding-policy.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  - Ease of national travel to the location
+  - Availability of lodging and accommodation
+  - Tourism in location
+- Applications may include additional information that may be important in the decision-making process for the Canadian Championship Selection Committee
+
+## Canadian Championship Selection Process
+
+- Once the Application Submission Process is complete, the Canadian Championship Selection Process will begin.
+- The review process **must** consist of communications between the Canadian Championship Selection Committee and a final vote to select the host of the Canadian Championship.
+  - The review period _may_ include meetings between the Canadian Championship Selection Committee.
```
Comment (sarahstrong314) [inline public/src/canadian-championship-bidding-policy.md:60]:
```suggestion
The review period _may_ include virtual meetings among the Canadian Championship Selection Committee.
```

File: public/src/canadian-championship-bidding-policy.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+  - Meetings _may_ be proposed with each organization team before voting has begun.
```
Comment (sarahstrong314) [inline public/src/canadian-championship-bidding-policy.md:61]:
```suggestion
Virtual meetings between the Canadian Championship Selection Committee and each organization team _may_ be proposed before voting has begun.
```

File: public/src/canadian-championship-bidding-policy.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+- The voting process **must** be conducted using a ranked ballot system.
+  - Each member of the Canadian Championship Selection Committee will receive a ballot listing all submitted applications.
+  - Committee members *must* rank the bids in order of preference, assigning a unique rank to each bid.
+    - 1 will be considered the highest preference, with each descending number representing a lower preference.
+  - Once all committee members have submitted their ballots, the votes will be tabulated. One bid *must* receive a majority of votes to win.
+    - In each round of tabulation, the bid with the fewest first-choice rankings will be eliminated, and its votes will be redistributed to the next-ranked bid on each ballot.
+    - This process continues until one bid receives a majority of votes, making it the preferred choice of the committee.
+    - In the case of a time, the bid with the highest number of second-choice rankings will be declared the winner. If a tie persists, the bid with the highest number of third-choice rankings will be declared the winner. This process will continue until the tie is broken.
```
Comment (kr-matthews) [inline public/src/canadian-championship-bidding-policy.md:70]:
```suggestion
    - In the case of a tie, the bid with the highest number of second-choice rankings will be declared the winner. If a tie persists, the bid with the highest number of third-choice rankings will be declared the winner. This process will continue until the tie is broken.
```

File: public/src/canadian-championship-bidding-policy.md (hunk 2 above)
Comment (kr-matthews) [reply public/src/canadian-championship-bidding-policy.md:34]: The document talks about the point person a lot, but this person only exists/is defined if the application is being submitted on behalf of a larger team. If an individual is submitting, then I assume all the requirements about the point person are intended to apply to that individual (such as "**must** be a member of Speedcubing Canada")?

File: public/src/canadian-championship-bidding-policy.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+- Once the Application Submission Process is complete, the Canadian Championship Selection Process will begin.
+- The review process **must** consist of communications between the Canadian Championship Selection Committee and a final vote to select the host of the Canadian Championship.
+  - The review period _may_ include meetings between the Canadian Championship Selection Committee.
+  - Meetings _may_ be proposed with each organization team before voting has begun.
+    - Modifications _may_ be made to an application if it is required by the Canadian Championship Selection Committee.
+- The voting process **must** be conducted using a ranked ballot system.
+  - Each member of the Canadian Championship Selection Committee will receive a ballot listing all submitted applications.
+  - Committee members *must* rank the bids in order of preference, assigning a unique rank to each bid.
+    - 1 will be considered the highest preference, with each descending number representing a lower preference.
```
Comment (kr-matthews) [inline public/src/canadian-championship-bidding-policy.md:66]: Does this type of detail belong in this document? How the ranked ballot system implements these requirements should be up irrelevant here?

File: public/src/canadian-championship-bidding-policy.md (hunk 5 above)
Comment (kr-matthews) [reply public/src/canadian-championship-bidding-policy.md:70]: What if the tie is never broken?

File: public/src/canadian-championship-bidding-policy.md (hunk 6 above)
Comment (tdmittens) [reply public/src/canadian-championship-bidding-policy.md:66]: Yeah, good point. I'll go ahead and acknowledge that we have a voting process, but the specifics aren't needed.

Comment (kr-matthews) [reply public/src/canadian-championship-bidding-policy.md]: I just meant the fact that 1 is considered highest is an implementation detail that doesn't matter. I liked all the other points.

File: public/src/canadian-championship-bidding-policy.md (hunk 2 above)
Comment (kr-matthews) [reply public/src/canadian-championship-bidding-policy.md]: ^ Was this noted? I'm still unclear.

File: public/src/canadian-championship-bidding-policy.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+Version: 1.0
+Updated: May 11, 2024
+
+# Canadian Championship Bidding Policy
+
+Uses of the words “must”, “must not”, “should”, “should not” and “may” match [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+Speedcubing Canada’s Board of Directors reserves the right to modify this document if a majority of not less than two-thirds (2/3) of Speedcubing Canada’s Board of Directors vote in favour of the modification.
+
+Canadian Championships _should_ should be conducted every 2 years, unless otherwise noted by Speedcubing Canada's Board of Directors.
```
Comment (bmetherall) [inline public/src/canadian-championship-bidding-policy.md:14]: Double should

File: public/src/canadian-championship-bidding-policy.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+- Every application _may_ include the following:
+  - Ease of national travel to the location
+  - Availability of lodging and accommodation
+  - Tourism in location
+- Applications may include additional information that may be important in the decision-making process for the Canadian Championship Selection Committee
+
+## Canadian Championship Selection Process
+
+- Once the Application Submission Process is complete, the Canadian Championship Selection Process will begin.
+- The review process **must** consist of communications between the Canadian Championship Selection Committee and a final vote to select the host of the Canadian Championship.
```
Comment (bmetherall) [inline public/src/canadian-championship-bidding-policy.md:59]: I find 'between' confusing. I suggest 'within'

**Review by JonEsparaz — APPROVED** [review]
Approving with the understanding that @tdmittens will address the comments in a follow-up PR, but allow us to get the process started.

**Pull request merged**